src/plugins/imagery/imageryViewProvider.js

```diff
@@ -0,0 +1,57 @@
+define([
+    './components/imageryViewLayout.vue',
+    'vue'
+], function (
+    ImageryViewLayoutComponent,
+    Vue
```

Comment:
Let's us ES6 imports.

Reply:
Also, the naming convention for at least the JS files has been UpperCamelCase with the exception of plugin.js (not sure why!). There has been some inconsistencies in the tcr branch with the new Vue components, but the majority of the files follow the UpperCamleCase convention. Please fix this file name as well as imageryViewLayout.vue.

Reply:
updated.